substrate/frame/bounties/src/lib.rs

```diff
@@ -885,6 +894,35 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		/// Poke the deposit reserved for creating a bounty proposal.
+		///
+		/// This can be used by accounts to possibly lower their locked amount.
```

Comment:
```suggestion
		/// This can be used by accounts to possibly change their reserved amount.
```

It can go either lower or higher right? and it is reserved, not locked.

Reply:
This was a suggestion from @ggwpez : [link](https://github.com/paritytech/polkadot-sdk/pull/7801#discussion_r1983373810)

I have no strong opinions either way. Will update unless there are more comments on this thread.